### PR TITLE
chore: streamline logging across modules

### PR DESCRIPTION
**Pull request by stijnmoreels**
Streamline logging and level across Powershell modules from `Arcus.Scripting.AzureActiveDirectory` to `Arcus.Scripting.LogicApps`, alphabetically.

Relates to #155

### CODE REVIEW

File: src/Arcus.Scripting.ActiveDirectory/Scripts/Add-AzADAppRoleAssignment.ps1 [hunk 1]
```diff
@@ -38,11 +38,11 @@ try {
         $adApplication.AppRole += $newRole
 
         Update-AzADApplication -ObjectId $adApplication.Id -AppRole $adApplication.AppRole
-        Write-Host "Added role '$Role' to Active Directory Application '$($adApplication.DisplayName)'" -ForegroundColor White
+        Write-Host "Added role '$Role' to Active Directory Application '$($adApplication.DisplayName)e'"
```
Comment (pim-simons) [inline src/Arcus.Scripting.ActiveDirectory/Scripts/Add-AzADAppRoleAssignment.ps1:41]:
```suggestion
        Write-Host "Added role '$Role' to Active Directory Application '$($adApplication.DisplayName)'"
```

File: src/Arcus.Scripting.LogicApps/Scripts/Disable-AzLogicAppsFromConfig.ps1 [hunk 2]
```diff
@@ -61,35 +61,35 @@ function ExecuteCheckType() {
...
                                 }
                             }
-                        }else{
-                            Write-Host("Found no more waiting or running runs for '$logicApp', executing stopType for Logic App")
+                        } else{
+                            Write-Verbose "Found no more waiting or running runs for Azure Logic App '$logicApp', executing stopType for Logic App"
                             ExecuteStopType -resourceGroupName $ResourceGroupName -LogicAppName $logicApp -stopType $batch.stopType
                         }                    
-                        Write-Host("> Check 'NoWaitingOrRunningRuns' executed successfully on '$logicApp'")   
+                        Write-Host "Check 'NoWaitingOrRunningRuns' executed successfully on Azure Logic APp '$logicApp'" -ForegroundColor Green
```
Comment (pim-simons) [inline src/Arcus.Scripting.LogicApps/Scripts/Disable-AzLogicAppsFromConfig.ps1:79]:
```suggestion
                        Write-Host "Check 'NoWaitingOrRunningRuns' executed successfully on Azure Logic App '$logicApp'" -ForegroundColor Green
```

**Review by pim-simons — CHANGES_REQUESTED** [review]
Looking good 👍🏻 
We should probably also update the docs as the logging has changed, for example https://scripting.arcus-azure.net/Features/powershell/azure-logic-apps#disable-an-azure-logic-app states:
``` powershell
PS> Disable-AzLogicApp -ResourceGroupName "rg-common-dev" -LogicAppName "rcv-shopping-order-sftp"
# Access-token and subscriptionId retrieved
# Attempting to disable rcv-shopping-order-sftp

# Successfully disabled rcv-shopping-order-sftp
```

And that is not correct anymore after this PR.

**Comment by stijnmoreels** [comment]
> We should probably also update the docs as the logging has changed,

Yes, I was thinking about that. Maybe we should not have the 'exact' logging in the docs as it would be rather hard/tedious to get them up-to-date all the time. Maybe we can only include a truncated version of information-like/successful messages?

File: src/Arcus.Scripting.ActiveDirectory/Scripts/Add-AzADAppRoleAssignment.ps1 (hunk 1 above)
Comment (stijnmoreels) [reply src/Arcus.Scripting.ActiveDirectory/Scripts/Add-AzADAppRoleAssignment.ps1]: Woops, what happened here 😅 , thx!

**Comment by pim-simons** [comment]
> > We should probably also update the docs as the logging has changed,
> 
> Yes, I was thinking about that. Maybe we should not have the 'exact' logging in the docs as it would be rather hard/tedious to get them up-to-date all the time. Maybe we can only include a truncated version of information-like/successful messages?

I get your point, but I honestly quite like the way we have it setup in the docs right now, it shows you exactly what to expect which I really like.

**Comment by stijnmoreels** [comment]
Now, it is updated with the full correct information/success messages.

**Review by pim-simons — APPROVED** [review]
Looks good to me 🥇

**Pull request merged**